extension/legacy/ChinaNewtabContentSearchParent.jsm

```diff
@@ -27,7 +27,12 @@ XPCOMUtils.defineLazyGetter(this, "ContentSearch", () => {
   const { ContentSearch } = ChromeUtils.import(contentSearchJSM);
   if (!ContentSearch.receiveMessage) {
     ContentSearch._reply = (browser, type, data) => {
-      if (browser.remoteType === "privilegedabout") {
+      if (
+        browser.remoteType === "privilegedabout" ||
+        // `about:privatebrowsing` is not loaded in `privilegedabout` process
+        // until Fx 86, see https://bugzil.la/1687359
+        browser.currentURI.prePath === "about:"
```

Comment:
Is there a reason you're not checking for about:private browsing here? Won't this load on all about pages?

Reply:
The `about:privatebrowsing` situation mentioned in the comment would happen on Fx 85 (or earlier versions), in more ordinary usage of Fx.

Checking `prePath` instead of `spec` is intended to cover a more edge case:

An user starts Fx with a previous version of china-newtab installed, and the pref `browser.tabs.remote.separatePrivilegedContentProcess` is flipped to `false` on the default branch during its startup. Then the extension is updated to this newer version, the pref will stay `false` before Fx shutdown.